packages/core/database/src/repairs/index.ts

```diff
@@ -0,0 +1,9 @@
+import { curry } from 'lodash/fp';
+import type { Database } from '..';
+import { removeOrphanMorphType as removeOrphanMorphTypeFunc } from './operations/remove-orphan-morph-types';
+
+export const createRepairManager = (db: Database) => {
+  return {
+    removeOrphanMorphType: curry(removeOrphanMorphTypeFunc)(db),
```

Comment:
just a thought I believe lodash curry messes up the typings in som ecases (might be when there are many params I don't recall. Did you notice anything ?

Reply:
I just checked where I added an asyncCurry elsewhere, and the problems are that it isn't aware of async functions, and it also provides ugly typings in general.

For now I'll add an internal util for asyncCurry similar to the other one, and then later on I think we should either add a @strapi/util to share between packages, or just start using Ramda 😆